src/bundle/Tests/ParamConverter/ContentTypeDraftParamConverterTest.php

```diff
@@ -65,4 +69,13 @@ public function testApplyWithWrongAttribute()
         $this->assertFalse($this->converter->apply($request, $config));
         $this->assertNull($request->attributes->get(self::PARAMETER_NAME));
     }
+
+    public function dataProvider(): array
```

Comment:
Those came from contentTypeId marked as mixed, right? If yes, not sure about those as we are moving to strict type those as `int` now.

also, 1000th PR! :tada:

Reply:
We test `string` here because param converter casts passed value to an integer

Reply:
Yup, for some reason I thought it is passed somewhere else, not to ParamConverter.